cmd/oci-runtime-tool/generate.go

```diff
@@ -34,6 +34,8 @@ var generateFlags = []cli.Flag{
 	cli.Uint64Flag{Name: "linux-cpu-period", Usage: "the CPU period to be used for hardcapping (in usecs)"},
 	cli.Uint64Flag{Name: "linux-cpu-quota", Usage: "the allowed CPU time in a given period (in usecs)"},
 	cli.StringFlag{Name: "linux-cpus", Usage: "CPUs to use within the cpuset (default is to use any CPU available)"},
+	cli.StringSliceFlag{Name: "linux-hugepage-limit-add", Usage: "add hugepage resource limit"},
+	cli.StringSliceFlag{Name: "linux-hugepage-limit-drop", Usage: "drp hugepage resource limit"},
```

Comment:
“drp” → “drop”.  fixed.
and the two options can be specified multiple times, so all modify from "limit" to "limits"